app-crypt/openpgp-keys-voikko/openpgp-keys-voikko-20201013.ebuild

```diff
@@ -0,0 +1,19 @@
+# Copyright 2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+DESCRIPTION="OpenPGP keys used to sign voikko* packages"
+HOMEPAGE="https://voikko.puimula.org/"
+SRC_URI="http://keys.gnupg.net/pks/lookup?op=get&search=0x3D309B604AE3942E -> 4AE3942E.asc"
+
+LICENSE="public-domain"
+SLOT="0"
+KEYWORDS="amd64 x86"
```

Comment:
Copy all keywords from the other ebuilds.

Reply:
I wanted to match KEYWORDS with voikko packages, but guess this won't hurt considering possible future keywordreq.

All done.